lint-staged.config.mjs

```diff
@@ -0,0 +1,4 @@
+export default {
+  '*.{ts,js,mjs,css}': ['prettier --write', 'eslint --fix'],
```

Comment:
i note that you’ve added mjs so made me think whether we want all of `[c|m]?js` and `[c|m]?ts`